sample-metadata/README.adoc

```diff
@@ -148,8 +156,18 @@ In the following example, only if the technical replicate column is provided, on
 | Sample 1          |    run 4   | label free sample | 2                            | 2                            | 000261_C05_P0001563_A00_B00K_F2_TR2.RAW
 |===
 
+NOTE: The comment[technical replicate] column is OPTIONAL and can be easily guess from the data structure if the column `comment[technical replicate]` is not provided. If the same sample has the fraction identifier repeated it can be interpreted as technical replicate. However, some submitters can make more clear that annotation.
 
-The "comment" columns in *SDRF* are included as a basic extensibility mechanism for local implementations. The name associated with the comment is included in square brackets in the column heading, and the value(s) entered in the body of the column. _comment_ columns could be used in various ways - to provide references to external files like raw files, or to include identifiers of objects in external systems.
+**Biological replicate**: parallel measurements of biologically distinct samples that capture biological variation, which may itself be a subject of study or a source of noise. Biological replicates address if and how widely the results of an experiment can be generalized. For example, repeating a particular assay with independently generated samples, individuals or samples derived from various cell types, tissue types, or organisms, to see if similar results can be observed. Context is critical, and appropriate biological replicates will indicate whether an experimental effect is sustainable under a different set of biological variables or an anomaly itself.
+
+NOTE: In SDRF proteomics biological replicates can be annotated using `characteristics[biological replicate]` but is OPTIONAL. This information can be also guess using the `factor value`
+and the `source name`, if two different source names (samples) are provided for the same factor value (condition) in the experiment they can be considered as biological replicates.
+
+Some examples with explicitly annotation of the biological replicates can be found here:
```

Comment:
This example annotates individuals, it is unclear what is replicated. I strongly recommend not using this terminology here.

Reply:
This has been removed from the current version. The columns are optional now.